roles/edpm_network_config/tasks/os_net_config.yml

```diff
@@ -26,12 +26,13 @@
     - name: Render overidden network config
       no_log: "{{ edpm_network_config_hide_sensitive_logs | bool }}"
       ansible.builtin.copy:
-        content: "{{ edpm_network_config_override | to_yaml }}"
```

Comment:
Isn't `content` is expected a string and `to_yaml` return one?  I  guess it also validates that.

Reply:
I know we discussed this on Slack. But for continuity here. It ended up printing the contents of the file in a single line with all of the `\n`'s in the string. 

As you suggested, `| from_yaml | to_yaml` does indeed fix that.